Dockerfile-alpine

```diff
@@ -0,0 +1,13 @@
+FROM golang:latest as builder
+
+RUN go get github.com/asobti/kube-monkey
+WORKDIR $GOPATH/src/github.com/asobti/kube-monkey
+#RUN make build
```

Comment:
Same comments still apply, we should use make instead. It's even installed!
```
$ docker run -it golang:latest sh
# make
make: *** No targets specified and no makefile found.  Stop.
```

Reply:
I found following error on dockerhub while building image
```
Makefile:37: recipe for target 'test' failed
```

Ref: https://hub.docker.com/r/nsidhaye/kube-monkey/builds/bylmem6tox6qxizhe3gaoqx/

Reply:
Ahh so it's the test that is failing because it expects the GOROOT to be set as `runtime.GOROOT() + "/lib/time/zoneinfo.zip"`

```
    --- FAIL: TestSuite/TestTimezone (0.00s)
    	config_test.go:54: 
    			Error Trace:	config_test.go:54
    			            				config.go:88
    			            				config_test.go:57
    			Error:      	Not equal: 
    			            	expected: "cannot find nolnexistent in zip file /usr/local/go/lib/time/zoneinfo.zip"
    			            	actual  : "cannot find nolnexistent in zip file /lib/time/zoneinfo.zip"
    			Test:       	TestSuite/TestTimezone
```

Reply:
I merged a fix for this earlier. See commit c186941

You probably just need to rebase

Reply:
Weird... doesn't this do a `go get` though?

Reply:
hmm, good point. Is `go get` maybe fetching the latest "release" and not HEAD?

Reply:
Able to test it. It's working...